### PR TITLE
Bumped page preload  to 20 (yoinked from Neko)

### LINKED ISSUE #544: [Feature Request] pre-load pages
### User Benefit
For smoother reading experience

### How should it work? 
Its pre-loads pages, similar feature to tachiyomiSY

### PR DESCRIPTION
**Pull request by joseph619**
will help with the upcoming...reader feature

Also closes #544

### CODE REVIEW

**Comment by joseph619** [comment]
@CarlosEsco hope it's fine :3

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/loader/HttpPageLoader.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package eu.kanade.tachiyomi.ui.reader.loader
 
 import android.graphics.BitmapFactory
+import com.elvishew.xlog.XLog
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/loader/HttpPageLoader.kt:4]: unused import added

**Review by Jays2Kings — COMMENTED** [review]
Change is fine but I'll add it as a setting instead of a straight change for every single source

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/loader/HttpPageLoader.kt (hunk 1 above)
Comment (joseph619) [reply app/src/main/java/eu/kanade/tachiyomi/ui/reader/loader/HttpPageLoader.kt:4]: removing the import

**Comment by joseph619** [comment]
> Change is fine but I'll add it as a setting instead of a straight change for every single source

Yeah that's fine by me 😁

**Comment by Jays2Kings** [comment]
Closing since it's been added in https://github.com/Jays2Kings/tachiyomiJ2K/commit/74826bc51bd94c1a57b4644ef895c3669ba29e99

**Pull request closed without merging**